build.gradle.kts

```diff
@@ -18,8 +18,8 @@ val camelVersion = "3.14.1"
 dependencies {
   annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
-  runtimeOnly("com.h2database:h2")
-  runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("com.h2database:h2:2.1.210")
```

Comment:
Could this be converted to testruntimeonly

Reply:
good idea